Good-Bids/goodbids
# [#519] Add Additional Nonprofit Fields

# Summary

This PR adds some additional fields to the Nonprofit Details form.

## Issues

* #519 

## Testing Instructions

1. Visit the Network Admin
2. Click Edit for a Nonprofit Site
3. Make sure field valuess can be set and changed.

## Screenshots

<img width="667" alt="Screenshot 2024-02-28 at 6 05 10 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/7a9534f1-51c3-4c2e-929b-12ea567c332a">

@clatwell Note: Super Admins can still edit NP fields, however BDP Admins won't be able to after they are verified.
